cesium/internal/unary/delete_test.go

```diff
@@ -21,7 +21,7 @@ import (
 
 var _ = Describe("Delete", Ordered, func() {
 	for fsName, fs := range fileSystems {
```

Comment:
Minor, but maybe you should rename the fs creator to `newFS` or something? Kinda confusing when you're assigning `fs := fs()`